docs/source/notebooks/temp-sense-gen/temp_sense_genCollab.ipynb

```diff
@@ -122,14 +122,16 @@
     "import pathlib\n",
     "import sys\n",
     "\n",
-    "!apt-get install -y ruby-full time build-essential\n",
-    "!apt install -f libqt4-designer libqt4-xml libqt4-sql libqt4-network libqtcore4 libqtgui4\n",
+    "!apt install -y time build-essential\n",
+    "!apt install qt5-default qttools5-dev libqt5xmlpatterns5-dev qtmultimedia5-dev libqt5multimediawidgets5 libqt5svg5-dev -y\n",
+    "!apt install ruby ruby-dev libz-dev python3-dev -y\n",
+    "!wget https://www.klayout.org/downloads/Ubuntu-20/klayout_0.28.3-1_amd64.deb\n",
```

Comment:
note: we have a package for klayout now too!
https://anaconda.org/LiteX-Hub/klayout/files

Reply:
@Jerry-lqx and @saicharan0112 can you update to use KLayout?

Reply:
@Jerry-lqx can you remove those lines and add `klayout` below yosys please?

Reply:
If I edit the raw file directly, I am afraid that it might break the notebook

Reply:
Yes, I'll make the changes